rts/lra/lra-test/basic/src/test/resources/META-INF/services/org.jboss.arquillian.core.spi.LoadableExtension

```diff
@@ -0,0 +1 @@
+io.narayana.lra.arquillian.extension.ArquillianLRATckExtension
```

Comment:
does basic tests needs only the `lra-archive-provider` or the `ArquillianLRATckExtension` as a whole. What about to create rather a new module named like `arquillian-extension` that would define the `LoadableExtension` class and other projects in `lra-test` module would depend on it as whole? That would mean extracting whole arq extension stuff from tck and move it to a separate module.